app/Libraries/BeatmapsetDiscussionReview.php

```diff
@@ -97,6 +101,22 @@ public static function create(Beatmapset $beatmapset, array $document, User $use
             BeatmapDiscussion::whereIn('id', $childIds)
                 ->update(['parent_id' => $review->getKey()]);
 
+            // handle disqualifications and the resetting of nominations
+            if ($problemPost) {
```

Comment:
This is for posting new review, but what about when updating an existing one creating new problem embed?

Reply:
Related issue: #6087.